frontend/src/pages/NewRunV2.test.tsx

```diff
@@ -266,30 +260,24 @@ describe('NewRunV2', () => {
   beforeEach(() => {});
 
   it('Fulfill default run value (start a new run)', async () => {
-    const getPipelineSpy = jest.spyOn(Apis.pipelineServiceApi, 'getPipeline');
+    const getPipelineSpy = jest.spyOn(Apis.pipelineServiceApiV2, 'getPipeline');
     getPipelineSpy.mockResolvedValue(ORIGINAL_TEST_PIPELINE);
-    const getPipelineVersionSpy = jest.spyOn(Apis.pipelineServiceApi, 'getPipelineVersion');
+    const getPipelineVersionSpy = jest.spyOn(Apis.pipelineServiceApiV2, 'getPipelineVersion');
     getPipelineVersionSpy.mockResolvedValue(ORIGINAL_TEST_PIPELINE_VERSION);
-    const getPipelineVersionTemplateSpy = jest.spyOn(
-      Apis.pipelineServiceApi,
-      'getPipelineVersionTemplate',
-    );
-    getPipelineVersionTemplateSpy.mockImplementation(() =>
-      Promise.resolve({ template: v2YamlTemplateString }),
-    );
+
     render(
       <CommonTestWrapper>
         <NewRunV2
           {...generatePropsNewRun()}
-          existingRunId='e0115ac1-0479-4194-a22d-01e65e09a32b'
+          existingRunId={null}
```

Comment:
What's the reason for this change?

Reply:
The `existingRunId` will exist only when `existingRun` is existing, so the previous one is incorrect although it didn't lead to failed tests.